src/ActiveLogin.Authentication.GrandId.AspNetCore/GrandIdAuthenticationBuilder.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Security.Authentication;
+using ActiveLogin.Authentication.GrandId.Api;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace ActiveLogin.Authentication.GrandId.AspNetCore
+{
+    public class GrandIdAuthenticationBuilder
+    {
+        public AuthenticationBuilder AuthenticationBuilder { get; }
+
+        private readonly List<Action<HttpClient>> _httpClientConfigurators = new List<Action<HttpClient>>();
+        private readonly List<Action<HttpClientHandler>> _httpClientHandlerConfigurators = new List<Action<HttpClientHandler>>();
+
+        public IServiceCollection Services;
```

Comment:
Not sure that `Services` should be exposed publicly at this level, it can be accessed via `AuthenticationBuilder.Services` anyway. Accessing via `AuthenticationBuilder` also gives a better hint that we are not in any way mixing service collections. 

If we for some reason want to keep it public, it should at least be `get` only (just like we do it for `AuthenticationBuilder`)   in my opinion.

Reply:
I agree. Fixed now.